liota-lite/MANIFEST.in

```diff
@@ -2,3 +2,5 @@ include *.md
 include *.txt
 
 recursive-include config *
+recursive-include packages/dev_disc *
+recursive-include packages/liotad *
```

Comment:
It seems you moved liotad.py, liotapkg.sh, cal_sha1sum.py, packages_auto.txt and README.md into packages/liotad folder. You only modified liota-lite/MANIFEST.in, but how about liota's MANIFEST.in?
Since it is still "recursive-include packages *", for liota, we will have packages/liotad also, right?

But how about liota.conf, where we have [PKG_CFG] section. 
"pkg_list = /usr/lib/liota/packages/packages_auto.txt" should be modified correspondingly.
How about "pkg_path = /usr/lib/liota/packages"? Will we still put user loaded packages under "packages" or under "packages/liotad"?

Reply:
The liota`s Manifest.in just require the packages folder path as it recursively includes all the directories present in it so there is no need of including the folder explicitly.

I`ll update the path in liota.conf for pkg_list, but packages will still remain in the folder "/usr/lib/liota/packages". I think we should also update the path for pkg_msg_pipe to "/usr/lib/liota/packages/liotad/package_messenger.fifo"

Reply:
Let me know your thoughts on updating the path for pkg_msg_pipe?

Reply:
yeah, pkg_msg_pipe can also be inside liotad folder.